.travis.yml

```diff
@@ -6,11 +6,13 @@ env:
   - MODE=main
 install:
     # Install runtime
-  - |-
-    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
-    sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
-    sudo apt-get update
-    sudo apt-get install dotnet-dev-1.0.0-rc4-004769 -y
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install 7.10.0
+  - npm install -g npm@'>=5.4.0'
+  - curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+  - sudo mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
+  - sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-trusty-prod trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
```

Comment:
try out `sudo apt-get install nodejs` specific version you want

Reply:
That wont work properly. To properly install a specific major version of nodejs you need to run a remote script like [https://deb.nodesource.com/setup_8.x](https://deb.nodesource.com/setup_8.x) and after that to run `sudo apt-get install nodejs`. 
We can do both, only that that way we won't have much granularity in the version used ( in case we care )